Source/WebCore/platform/graphics/FontCascadeDescription.cpp

```diff
@@ -43,13 +43,14 @@ struct SameSizeAsFontCascadeDescription {
     AtomString string2;
     int16_t fontSelectionRequest[3];
     float size;
+    std::optional<float> sizeAdjust;
     unsigned bitfields1;
     unsigned bitfields2 : 22;
     void* array;
     float size2;
     unsigned bitfields3 : 10;
 };
-
+template<int s> struct Wow;
```

Comment:
???

Reply:
That's actually funny. It was for investigating a bit alignment issue. Forgot to remove everything, sorry.